browser/playlist/playlist_tab_helper.cc

```diff
@@ -82,6 +85,29 @@ base::WeakPtr<PlaylistTabHelper> PlaylistTabHelper::GetWeakPtr() {
   return weak_ptr_factory_.GetWeakPtr();
 }
 
+std::u16string PlaylistTabHelper::GetSavedFolderName() {
+  CHECK(saved_items_.size()) << "Caller should check if there are saved items";
+  if (const auto& parents = saved_items_.front()->parents;
+      parents.empty() || parents.size() >= 2) {
+    return u"Playlist";
+  }
+
+  const auto parent_id = saved_items_.front()->parents.front();
+  if (std::any_of(saved_items_.begin() + 1, saved_items_.end(),
```

Comment:
it would be good to have comments something like `use saved folder name when all saved items have same parent.`

Reply:
Great. Added comment on top of the function.